djweb/dj_comp_hist/metadata_parser.py

```diff
@@ -56,15 +56,34 @@ def populate_from_metadata(file_name=None):
     print(f'Added {count_added} documents from {file_name}. Skipped {count_skipped} documents '
           f'because of incomplete metadata. Invalid: {count_invalid}')
 
+
     db = sqlite3.connect(f"{Path(DJWEB_PATH.parent, 'db.sqlite3')}")
     cursor = db.cursor()
+
+    # Create full text search table for document text, title, author, recipient, cced
     # delete old table if exists before populating full text search table.
     cursor.execute('DROP TABLE IF EXISTS doc_fts;')
-    cursor.execute('create virtual table doc_fts using FTS4(id, title, text);')
-    cursor.execute('''INSERT INTO doc_fts(id, title, text) 
-                                                SELECT id, title, text 
-                                                FROM dj_comp_hist_document;''')
-    db.commit()
+    cursor.execute('''CREATE VIRTUAL TABLE doc_fts USING FTS4(id, title, text, author, recipient, 
+                                                                                       cced);''')
+    for d in Document.objects.all():
+        # join together all author names. By default, first and last name are joined by an
+        # underscore -> replace with a space
+        author = " ".join([p['name'].replace('_', ' ') for p in d.get_person_list('authors')])
+        recipient = " ".join([p['name'].replace('_', ' ') for p in d.get_person_list('recipients')])
+        cced = " ".join([p['name'].replace('_', ' ') for p in d.get_person_list('cceds')])
+
+        # Technically, sqlite expects quotation marks and apostrophes to be escaped by doubling
```

Comment:
There were escaping problems in some of the other fields, so I'm applying this same removal of ' and " to all the fields as a temporary workaround.